filebeat/tests/system/test_tcp_tls.py

```diff
@@ -50,7 +50,7 @@ def test_tcp_over_tls_and_verify_valid_server_without_mutual_auth(self):
 """
         config = {
             "host": "127.0.0.1",
-            "port": 8080,
+            "port": 8082,
```

Comment:
What is this talking to that the port number can change without needing to update anything else?

Reply:
Could you just make this port 0 to request an unallocated port?

Reply:
I had some of those tests failing because the port was already in use. There is a small chance this was due some Filebeat process that did not get killed on a test that error-ed. I can revert those changes and test it again.

Reply:
I updated the PR description with more details.

Reply:
Ah, you just need these test cases not to collide.

Reply:
Perhaps put a comment about that directly in this file